Calculator/Calculator/Controller/MainViewController.swift

```diff
@@ -0,0 +1,185 @@
+    @IBOutlet private weak var formulaListScrollView: UIScrollView!
+    @IBOutlet private weak var formulaListStackView: UIStackView!
+    
+    private var isReset: Bool = false
+    private var isZero: Bool = true
+    
+    private var operatorValue: String {
+        get {
+            return operatorLabel.text ?? CalculatorNamespace.Empty
+        }
+        set(newOperator) {
+            operatorLabel.text = newOperator
+        }
+    }
+    
+    private var operandValue: String {
+        get {
+            return operandLabel.text ?? CalculatorNamespace.Zero
+        }
+        set(newOperand) {
+            operandLabel.text = newOperand
+        }
+    }
+    
+    private var currentLabelValues: LabelValues {
+        get {
+            return (operandValue: operandValue, operatorValue: operatorValue)
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        deleteAllFormulaListStackView()
+        setUpLabelValues(LabelParser.getDefaultLabelValues())
+    }
+    
+    private func setUpLabelValues(_ status: LabelValues) {
+        self.operandValue = status.operandValue
+        self.operatorValue = status.operatorValue
+    }
+    
+    private func deleteAllFormulaListStackView() {
+        formulaListStackView.arrangedSubviews.forEach { stack in
+            stack.removeFromSuperview()
+        }
+    }
+    
+    @IBAction private func touchUpButton(_ sender: UIButton) {
+        guard let senderTitle = sender.currentTitle else { return }
+        let buttonType: ButtonType = ButtonType.getType(senderTitle)
+        
+        setIsZero(senderTitle: senderTitle)
+
+        switch buttonType {
+        case .equal:
+            touchUpEqualButton()
+            return
+        case .operators:
+            guard operandValue != CalculatorNamespace.NaN else { return }
+            touchUpOperatorButton()
+        case .allClear:
+            deleteAllFormulaListStackView()
+        case .numbers, .doubleZero, .dot:
+            setIsZero(senderTitle: senderTitle)
+            touchUpNumbersButton()
+        default:
+            break
+        }
+        let labelValues: LabelValues = LabelParser
+            .parseLabelValues(button: buttonType,
+                              buttonTitle: senderTitle,
+                              labelValues: currentLabelValues,
+                              isReset: isReset)
+        setUpLabelValues(labelValues)
+        isReset = buttonType == .signToggle ? isReset : false
+    }
+
+    private func setIsZero(senderTitle: String) {
+        if senderTitle == CalculatorNamespace.Zero ||
+           operandValue == CalculatorNamespace.Zero {
+            isZero = true
+        }
+        if isZero && operandValue != CalculatorNamespace.Zero {
+            if operatorValue != CalculatorNamespace.Empty {
+                addNewFormulaStackView(LabelParser.getDefaultLabelValues())
+            }
+            isZero = false
+        }
+    }
```

Comment:
```suggestion
    private func setIsZero(senderTitle: String) {
        if senderTitle == CalculatorNamespace.Zero ||
           operandValue == CalculatorNamespace.Zero {
            isZero = true
        }
        
        if isZero && operandValue != CalculatorNamespace.Zero {
            if operatorValue != CalculatorNamespace.Empty {
                addNewFormulaStackView(LabelParser.getDefaultLabelValues())
            }
            
            isZero = false
        }
    }
```
간혹 코드에 `조건문` 혹은 `guard문`을 개행없이 구현하시는 경우가 있으신데, 저는 이런 컨벤션이 가독성에 좋다고 생각이 듭니다!

Reply:
개행을 잊고 진행한 부분들이 있었네요ㅠㅠ! 수정 적용해 두겠습니다!